tests/components/smtp/test_notify.py

```diff
@@ -139,3 +68,83 @@ async def test_reload_notify(hass):
 
 def _get_fixtures_base_path():
     return path.dirname(path.dirname(path.dirname(__file__)))
+
+
+@pytest.fixture
+def message():
+    """Return MockSMTP object with test data."""
+    mailer = MockSMTP(
+        "localhost",
+        25,
+        5,
+        "test@test.com",
+        1,
+        "testuser",
+        "testpass",
+        ["recip1@example.com", "testrecip@test.com"],
+        "Home Assistant",
+        0,
+    )
+    yield mailer
+
+
+html = """
+        <!DOCTYPE html>
+        <html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+            <head><meta charset="UTF-8"></head>
+            <body>
+              <div>
+                <h1>Intruder alert at apartment!!</h1>
+              </div>
+              <div>
+                <img alt="tests/testing_config/media/test.jpg" src="cid:tests/testing_config/media/test.jpg"/>
+              </div>
+            </body>
+        </html>"""
+
+email_data = [
+    ("Test msg"),
+    ("Test msg", {"images": ["tests/testing_config/media/test.jpg"]}),
+    ("Test msg", {"html": html, "images": ["tests/testing_config/media/test.jpg"]}),
+    ("Test msg", {"html": html, "images": ["test.jpg"]}),
+    ("Test msg", {"html": html, "images": ["tests/testing_config/media/test.pdf"]}),
+]
+
+
+@pytest.mark.parametrize(
+    "email_data",
+    [email_data[0], email_data[1], email_data[2], email_data[3], email_data[4]],
```

Comment:
```suggestion
    EMAIL_DATA,
```

Reply:
Hi @MartinHjelmare, I don't think this is correct.
we can change Line 116 to 
`[EMAIL_DATA[0], EMAIL_DATA[1], EMAIL_DATA[2], EMAIL_DATA[3], EMAIL_DATA[4]],`

because Parametrize work in a for loop format and takes it each of these data from array for checking every case of the test. 
Waiting for your opinion here.

Reply:
It's the same:
```
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: EMAIL_DATA = [
   ...:     ("Test msg"),
   ...:     ("Test msg", {"images": ["tests/testing_config/media/test.jpg"]}),
   ...:     ("Test msg", {"html": "html", "images": ["tests/testing_config/media/test.jpg"]}),
   ...:     ("Test msg", {"html": "html", "images": ["test.jpg"]}),
   ...:     ("Test msg", {"html": "html", "images": ["tests/testing_config/media/test.pdf"]}),
   ...: ]

In [2]: EMAIL_DATA == [EMAIL_DATA[0], EMAIL_DATA[1], EMAIL_DATA[2], EMAIL_DATA[3], EMAIL_DATA[4]]
Out[2]: True
```

Reply:
Thanks for the explanation. Fixed this. Will submit in the next push.